rollup.config.js

```diff
@@ -1,63 +1,50 @@
-'use strict';
+"use strict";
```

Comment:
single quotes preferred.
This repo needs prettier, filed https://github.com/salesforce/observable-membrane/issues/79

Reply:
Even better, now that all JavaScript files are modules, they run out of the box in strict mode.